backend/api/views.py

```diff
@@ -359,15 +353,6 @@ def create_checkout_session(self, request, pk=None):
                     "quantity": purchase_item.get("quantity"),
                 }
                 to_send_stripe.append(a)
-                history = History_of_Item.objects.create(
```

Comment:
Delete this because we only want to change quantity if the payment goes through

Reply:
Makes sense to add a conditional that will create the history record when the payment goes through.

Reply:
o maybe. should we add a category? Even then, it should go in the "purchase_item" endpoint rather than the create_checkout_session. All this endpoint should do is create stripe instance for them

Reply:
hmmmm i see. i imagine stripe doesn't offer mock-related webhooks? if so, we should probably bring this up during our TA-demo so that they're fully aware of this situation.

Reply:
I mean, what we're doing now essentially does the same thing. the purchase_items endpoint confirms that a session was completed